modules/apps/analytics/analytics-dxp-entity-rest-api/src/main/java/com/liferay/analytics/dxp/entity/rest/dto/v1_0/DXPEntity.java

```diff
@@ -206,6 +206,30 @@ public boolean equals(Object object) {
 		return Objects.equals(toString(), dxpEntity.toString());
 	}
 
+	public Object getValue(String propertyName) {
```

Comment:
I think It does not make sense to generate this method for the DTOs that do not have any x-json-map property, at least by now.

For our use case, this method has to be generated only for the cases where x-json-map is true